docs/deployment-process/operations-runbooks/index.md

```diff
@@ -87,8 +84,7 @@ Record the HTTP requests made by the Octopus UI to see how we build your runbook
 
 For this early access feature, there are a number of things we decided to not include initially (so we could get the feature in your hands sooner). Some known limitations that are not currently supported are:
 
-- Prompted variables (if your project includes prompted variables, they will be ignored for runbooks).
-- Variable scoping to runbooks (there's no way to scope variables between your project's deployment process vs runbook processes).
+- There is no way to scope to a runbook action.
```

Comment:
Again, I know technically it's an action scope, but for customers, they think in terms of steps. So either step/action or just step would be more user-friendly reading.